### PR TITLE
Feature - Config API show list menu photos

### PR DESCRIPTION
**Pull request by at-hanguyen3**
- [x] Config API show list menu photos

### CODE REVIEW

**Comment by at-lamle** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controller/Detail/DetailViewModel.swift [hunk 1]
```diff
@@ -75,6 +71,10 @@ final class DetailViewModel: ViewModel {
         return comments.count
     }
 
+    func getIDForShowListMenu() -> Restaurant {
```
Comment (at-hainguyen3) [inline MyApp/View/Controller/Detail/DetailViewModel.swift]: Get id mà sao lại trả về model

File: MyApp/View/Controller/Detail/DetailViewModel.swift [hunk 2]
```diff
@@ -17,7 +17,7 @@ final class DetailViewModel: ViewModel {
     var restaurant = Restaurant()
     var comments: [Comment] = []
     var menus: [Food] = []
-    var venueId = ""
+    var venueId = "534cf4a9498ea2812a78f14c"
```
Comment (at-hainguyen3) [inline MyApp/View/Controller/Detail/DetailViewModel.swift]: Reset

File: MyApp/View/Controller/ShowListMenu/ListMenu/ListFoodsViewController.swift [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+//  ListFoodsViewController.swift
+//  MyApp
+//
+//  Created by PCI0010 on 3/6/19.
+//  Copyright © 2019 Asian Tech Co., Ltd. All rights reserved.
+//
+
+import UIKit
+
+class ListFoodsViewController: UIViewController {
```
Comment (at-bienle) [inline MyApp/View/Controller/ShowListMenu/ListMenu/ListFoodsViewController.swift]: final

File: MyApp/View/Controller/ShowListMenu/ListPhotos/Cell/AlbumPhotoCellViewModel.swift [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+import Foundation
+
+final class AlbumPhotoCellViewModel {
+
+    // MARK: - Properties
+    var thumbnail: String = ""
+
+    // MARK: - Init
+    init(photos: Photo) {
```
Comment (at-bienle) [inline MyApp/View/Controller/ShowListMenu/ListPhotos/Cell/AlbumPhotoCellViewModel.swift]: `photo`, not photos

File: MyApp/View/Controller/ShowListMenu/ListPhotos/Cell/AlbumPhotoCollectionViewCell.swift [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+//  AlbumPhotoCollectionViewCell.swift
+//  MyApp
+//
+//  Created by PCI0010 on 3/6/19.
+//  Copyright © 2019 Asian Tech Co., Ltd. All rights reserved.
+//
+
+import UIKit
+
+class AlbumPhotoCollectionViewCell: UICollectionViewCell {
```
Comment (at-bienle) [inline MyApp/View/Controller/ShowListMenu/ListPhotos/Cell/AlbumPhotoCollectionViewCell.swift]: final

File: MyApp/View/Controller/ShowListMenu/ListPhotos/ListAlbumViewController.swift [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+        cell.viewModel = vm
+        return cell
+    }
+}
+
+// MARK: - UICollectionViewDelegate
+extension ListAlbumViewController: UICollectionViewDelegate {
+
+    func collectionView(_ collectionView: UICollectionView, willDisplay cell: UICollectionViewCell, forItemAt indexPath: IndexPath) {
+        if indexPath.row == viewModel.photos.count - 6 {
```
Comment (at-bienle) [inline MyApp/View/Controller/ShowListMenu/ListPhotos/ListAlbumViewController.swift]: Don`t try to access data directly form VIEW

File: MyApp/View/Controller/ShowListMenu/MenuViewController.swift [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+//  Created by PCI0010 on 3/6/19.
+//  Copyright © 2019 Asian Tech Co., Ltd. All rights reserved.
+//
+
+import UIKit
+
+final class MenuViewController: ViewController {
+
+    // MARK: - Outlet
+    @IBOutlet private weak var viewContaint: UIView!
```
Comment (at-bienle) [inline MyApp/View/Controller/ShowListMenu/MenuViewController.swift]: Rename, what is containt?

File: MyApp/View/Controller/ShowListMenu/MenuViewController.swift [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+        let vc = arrayVC[type.rawValue]
+        guard let pageVC = pageVC, vc != pageVC.viewControllers?.first else { return }
+        pageVC.setViewControllers([vc],
+                                  direction: type.direction,
+                                  animated: false,
+                                  completion: nil)
+    }
+
+    // MARK: - IBAction
+    @IBAction func tabButtonTouchUpInside(_ sender: UIButton) {
```
Comment (at-bienle) [inline MyApp/View/Controller/ShowListMenu/MenuViewController.swift]: private

File: MyApp/View/Controller/ShowListMenu/MenuViewController.swift [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+        guard let type = TypeChildPageViewController(rawValue: sender.tag) else { return }
+        switchingPageViewController(type: type)
+    }
+}
+
+// MARK: - UIPageViewControllerDataSource
+extension MenuViewController: UIPageViewControllerDataSource {
+    func pageViewController(_ pageViewController: UIPageViewController, viewControllerBefore viewController: UIViewController) -> UIViewController? {
```
Comment (at-bienle) [inline MyApp/View/Controller/ShowListMenu/MenuViewController.swift]: Add line spacing

**Pull request merged**